tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/OrderingPartyControllerTests.cs

```diff
@@ -79,30 +78,39 @@ public static void Constructor_VerifyGuardClauses()
         [Test]
         [InMemoryDbAutoData]
         public static async Task UpdateAsync_OrderDoesNotExist_ReturnsNotFound(
+            [Frozen] Mock<IOrderingPartyService> service,
             CallOffId callOffId,
             OrderingPartyModel model,
             OrderingPartyController controller)
         {
+            service.Setup(o => o.GetOrder(callOffId)).ReturnsAsync((Order)null);
+
             var response = await controller.UpdateAsync(callOffId, model);
+            service.Verify(o => o.GetOrder(callOffId));
```

Comment:
As above, I don't think we need to `Verify` here as the test covers this as part of its assertion.

If you think it's worthwhile verifying somewhere that `GetAsync` invokes `GetOrder`, I'd recommend adding a separate test for that.

Reply:
Removed `service.Verify()`
And added new Unit test method to test `GetOrder` Invoke